.pyproject_generation/pyproject_custom.toml

```
@@ -4,10 +4,10 @@ name = "my_microservice"
 version = "0.1.0"
 description = "My-Microservice - a short description"
 dependencies = [
-    "typer >= 0.9.0",
-    "ghga-service-commons[api] >= 1.2.0",
-    "ghga-event-schemas >= 1.0.0",
-    "hexkit[akafka,s3,mongodb] >= 1.1.0"
+    "typer >= 0.12",
+    "ghga-service-commons[api] >= 1.2",
+    "ghga-event-schemas >= 2.0",
```

```suggestion
    "ghga-event-schemas >= 3.0",
```
2.0 was based on wrong assumptions wrt the PCS deletion path, 3.0 basically reverted events schemas to 1.0. Might lead to some issues if this somehow actually got resolved as 2.0